op25/gr-op25_repeater/apps/tk_smartnet.py

```diff
@@ -28,6 +28,7 @@
 from log_ts import log_ts
 from collections import deque
 from gnuradio import gr
+from builtins import bytes
```

Comment:
This requires the user to `pip install future` - is there a way to do that automatically, or should we just add it to the install script or instructions? I imagine that package will get more use as more Python 3 support is added.